config.go

```diff
@@ -171,7 +173,7 @@ func (c *srcfileConfig) apply() error {
 		}
 		config.GOPATH = strings.Join(dirs, string(filepath.ListSeparator))
 
-		buildContext.GOPATH += string(filepath.ListSeparator) + config.GOPATH
+		buildContext.GOPATH = config.GOPATH + string(filepath.ListSeparator) + buildContext.GOPATH
```

Comment:
This change makes it so the the GOPATH from config gets higher priority, is that right?